site/content/how-to/data-plane-configuration.md

```diff
@@ -112,3 +112,51 @@ Status:
 ```
 
 If everything is valid, the `ResolvedRefs` condition should be `True`. Otherwise, you will see an `InvalidParameters` condition in the status.
+
+## Configure the Data Plane Log Level
+
+You can use the `NginxProxy` resource to dynamically configure the Data Plane Log Level.
+
+The following command creates a basic `NginxProxy` configuration that sets the log level to `warn` instead of the default value of `info`:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.nginx.org/v1alpha1
+kind: NginxProxy
+metadata:
+  name: ngf-proxy-config
+spec:
+  logging:
+    errorlevel: warn
+EOF
+```
+
+After attaching the NginxProxy to the GatewayClass, the log level of the data plane will be updated to `warn`.
+
+To view the full list of supported log levels, see the `NginxProxy spec` in the [API reference]({{< relref "reference/api.md" >}})
+
+{{< note >}}For `debug` logging to work, NGINX needs to be built with `--with-debug` or "in debug mode". For more information visit the official NGINX documentation
+["A debugging log"](https://nginx.org/en/docs/debugging_log.html). NGINX Gateway Fabric can easily
+be [run with NGINX in debug mode](#run-nginx-gateway-fabric-with-nginx-in-debug-mode) upon startup through the addition
+of a few arguments. {{</ note >}}
+
+## Run NGINX Gateway Fabric with NGINX in debug mode
```

Comment:
from @sjberman 

> Hm, this feels like it should be somewhere else...troubleshooting guide maybe?

Do you have any thoughts on this @ADubhlaoich ?

Reply:
I think it would be good to keep the debug instructions here. If I'm trying to change the log level I would likely come to this guide first to see how to do that. Maybe we can link to this header in the troubleshooting guide?

Reply:
Maybe we make it a subsection to the logging section then. Feels weird to have this as a top level header in a doc that is focused on updating nginx config using the CRD.

Reply:
made the section into a subsection of the previous, will await docs input